selenium/src/main/java/com/deque/html/axecore/extensions/WebDriverExtensions.java

```diff
@@ -12,7 +12,7 @@
 
 package com.deque.html.axecore.extensions;
 
-import com.deque.html.axecore.results.Results;
+import com.deque.html.axecore.results.AxeResults;

```

Comment:
This would be considered a breaking change, we'll need to to be reverted back to `Results`

Reply:
Reverted all instances of Axe Results to Results in selenium module